x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/onboarding/hover_for_explanation_tooltip.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import type { EuiToolTipProps } from '@elastic/eui';
+import { EuiToolTip, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+type HoverForExplanationTooltipProps = {
+  children: React.ReactNode;
+} & Partial<Omit<EuiToolTipProps, 'children'>>;
```

Comment:
just wondering, why remove children but add them ourselves? can't we just use `EuiToolTipProps`?

Reply:
true, I had added an workaround as initially hit type errors because it didn't recognized the default values, but it actually works now by using  `Partial<EuiToolTipProps>`, good call!